src/skins/vector/css/matrix-react-sdk/structures/RoomView.css

```diff
@@ -202,6 +202,25 @@ hr.mx_RoomView_myReadMarker {
     width: 100%;
     -webkit-flex: 0 0 auto;
     flex: 0 0 auto;
+
+    max-height: 0px;
+    background-color: #fff;
```

Comment:
I think this might have to be done as a separate PR because this was branched prior to scss stuff.